packages/admin-sdk/src/ui/menu/index.ts

```diff
@@ -1,7 +1,12 @@
 import { createSender } from '../../channel';
 
+export const collapseMenu = createSender('menuCollapse', {});
```

Comment:
Why the empty object for baseMessageOptions?

Reply:
I just got rid of it